lib/msal-core/src/Configuration.ts

```diff
@@ -63,6 +65,7 @@ export type SystemOptions = {
   logger?: Logger;
   loadFrameTimeout?: number;
   tokenRenewalOffsetSeconds?: number;
+  waitTimeToLoadFrame?: number;
```

Comment:
So the naming is a little confusing here with load, frame, and time. I think the usage here is a little different.

loadFrameTimeout is actually the time it will wait to load the frame. This parameter is being used as the timeout for frame navigation, so you could call it silentAuthTimeout or something. I think the name of the function loadFrame is probably misleading as well, we could change it to navigateFrame instead.

Reply:
👍 I feel the confusion too. Changed it now.